lib/plurimath/math/function/ternary_function.rb

```diff
@@ -51,6 +51,15 @@ def to_html
           "#{first_value}#{second_value}#{third_value}"
         end
 
+        def to_math_zone(spacing, last = false, indent = true)
+          constant = self.class::FUNCTION
```

Comment:
`constant` is a very generic name and does not make it clear why this is used. Try to use appropriate variable names. 
May be we should move the logic to class methods something like `self.class.first_parameter_name` etc